code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -213,7 +213,7 @@
 	inhand_y_dimension = 32
 	mag_type = /obj/item/ammo_box/magazine/internal/boltaction
 	fire_sound = 'sound/weapons/gunshots/gunshot_rifle.ogg'
-	var/bolt_open = 0
+	bolt_open = FALSE
```

Comment:
Unless it gets changed somewhere between here and `/obj/item/gun`, this assignment is redundant.

Reply:
Resolved. Thanks!